svthalia/concrexit
# Add promotion model

### Summary
Adds a new app for event organizers to request promotion material/messages. 

### How to test
1. Create promo requests for events via the events admin
2. Create promo requests via the 'regular' admin
3. Emails should be sent after creating a request
4. No emails are sent on updating an event (might be added later if desirable)
